### PR TITLE
New package: novprog-3.1.7.

### PR DESCRIPTION
**Pull request by rtiangha**
From the author of Focuswriter, NovProg is a tool to graph your progress while writing a NaNoWriMo style novel.

https://gottcode.org/novprog/

Builds on amd64, i686, armv7l and aarch64, tested on amd64.

### CODE REVIEW

**Comment by ericonr** [comment]
Squash the commits into a single one per package :)

File: srcpkgs/novprog/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'novprog'
+pkgname=novprog
+version=3.1.7
+revision=1
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake"
+makedepends="qt5-widgets qt5-tools-devel"
+short_desc="Wordcount tracking program"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/novprog/template:9]: It's usually expected that if you're making a PR for a new package, you will also maintain it. Keeping it up to date and potentially taking a look at issues people have, if any.

Comment (the-maldridge) [reply srcpkgs/novprog/template:9]: Actually its required.  Packages are not accepted pre-orphaned.

Comment (rtiangha) [reply srcpkgs/novprog/template:9]: Sure, I can do that. I'll add my contact info then.

**Pull request closed without merging**